TravelProject/TravelProject/Program.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+
+namespace TravelProject
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Please fill departure country: ");
+            var _departureCountry = Console.ReadLine();
```

Comment:
local variables names should not start with "_" prefix, it is used for private class fields. Local variables naming should follow the relu: var someUsefulvariable;